docs: cleanup in the PDE section related to PPD-913 (#1145)
spectrocloud/librarium: content/docs/04.5-devx/02-enterprise-user.md

# Overview

> You can use the Palette Develop Experience (PDE) to serve two common use cases: enterprise developers and individual application authors who want to deploy a containerized application without worrying about infrastructure overhead. 

Reviewer: ```suggestion
You can use the Palette Developer Experience (PDE) to serve two common use cases: enterprise developers and individual application authors who want to deploy a containerized application without worrying about infrastructure overhead. 
```

Refer to the [Enterprise Users](#enterprise-users) section to learn more about enabling PDE for a large set of downstream users. If you are an individual application author, check out the [Individual Application Author](#application-authors) section to get started.

# Enterprise Users

To enable PDE for downstream users, start by reviewing the following resources.

1. Understand the difference between [Cluster Mode and App Mode](/introduction/palette-modes).

2. Create and Manage the [Cluster Group](/clusters/cluster-groups).

3. Allocate [User Quota](/devx/resource-quota).

The next set of references are not required but good for Palette administrators to be aware of.

- [Enable Disk Backup on Virtual Clusters](/clusters/cluster-groups/cluster-group-backups).

- [Set Up Ingress for a Cluster Group](/clusters/cluster-groups/ingress-cluster-group).

- [Pause and Resume Palette Virtual Clusters](/devx/palette-virtual-clusters/pause-restore-virtual-clusters).

- [Resize Palette Virtual Clusters](/devx/palette-virtual-clusters/resize-virtual-clusters).

# Application Authors

> Use PDE and deploy your containerized applications to Palette. Palette offers a free-tier offering of PDE that you can use to get started. Create your application profiles and deploy your applications to Palette in no time.

Reviewer: ```suggestion
Use PDE to deploy your containerized applications to Palette. Leverage Palette's free-tier offering of PDE to get started. Create your application profiles and deploy your applications to Palette in no time.
```
